From 4096d03e117bd6cfff5a9f94cbffec2bd0712f26 Mon Sep 17 00:00:00 2001
From: Pantelis Antoniou <panto@antoniou-consulting.com>
Date: Thu, 10 Jan 2013 16:22:59 +0200
Subject: [PATCH 050/155] am33xx.dtsi: Make the MUSB not crash on load

Two entries for the same musb instance end up with a crash.
Remove the duplicate, but it still doesn't work.

Merged by Jason Kridner <jdk@ti.com> against updated musb-cppi41 patches.
---
 arch/arm/boot/dts/am33xx.dtsi |   37 -------------------------------------
 1 files changed, 0 insertions(+), 37 deletions(-)

diff --git a/arch/arm/boot/dts/am33xx.dtsi b/arch/arm/boot/dts/am33xx.dtsi
index 1fe62f4..27cfeb4 100644
--- a/arch/arm/boot/dts/am33xx.dtsi
+++ b/arch/arm/boot/dts/am33xx.dtsi
@@ -291,11 +291,6 @@
 			usb1-phy = <&usb1_phy>;
 		};
 
-		rtc {
-			compatible = "ti,da830-rtc";
-			ti,hwmods = "rtc";
-		};
-
 		spi0: spi@48030000 {
 			compatible = "ti,omap4-mcspi";
 			ti,hwmods = "spi0";
@@ -505,38 +500,6 @@
 			ti,hwmods = "rtc";
 		};
 
-		spi0: spi@48030000 {
-			compatible = "ti,omap4-mcspi";
-			#address-cells = <1>;
-			#size-cells = <0>;
-			reg = <0x48030000 0x400>;
-			interrupt = <65>;
-			ti,spi-num-cs = <2>;
-			ti,hwmods = "spi0";
-			dmas = <&edma 16
-				&edma 17
-				&edma 18
-				&edma 19>;
-			dma-names = "tx0", "rx0", "tx1", "rx1";
-			status = "disabled";
-		};
-
-		spi1: spi@481a0000 {
-			compatible = "ti,omap4-mcspi";
-			#address-cells = <1>;
-			#size-cells = <0>;
-			reg = <0x481a0000 0x400>;
-			interrupt = <125>;
-			ti,spi-num-cs = <2>;
-			ti,hwmods = "spi1";
-			dmas = <&edma 42
-				&edma 43
-				&edma 44
-				&edma 45>;
-			dma-names = "tx0", "rx0", "tx1", "rx1";
-			status = "disabled";
-		};
-
 		usb@47400000 {
 			compatible = "ti,musb-am33xx";
 			reg = <0x47400000 0x1000	/* usbss */
-- 
1.7.8.6

